app/page.test.tsx

```diff
@@ -2,10 +2,14 @@ import '@testing-library/jest-dom'
 import { render, screen } from '@testing-library/react'
 import Main from './page'
 
+jest.mock('next/navigation', () => ({
+  useRouter: jest.fn(),
+}))
```

Comment:
I don't think we need to mock the router here.

Reply:
If we are not mocking this, getting 'invariant expected app router to be mounted' error.